apis/twitter.js

```diff
@@ -0,0 +1,42 @@
+import Twit from 'twit';
+import getObjProperty from 'lodash.get';
+
+const {
+    TWITTER_ACCESS_SECRET,
+    TWITTER_ACCESS_TOKEN,
+    TWITTER_CONSUMER_KEY,
+    TWITTER_CONSUMER_SECRET
+} = process.env;
+
+const Twitter = new Twit({
```

Comment:
can we switch this to the [application context](https://github.com/ttezel/twit#twit-api) form? at some point I do want to allow GraphQLHub to handle per-user type of auth tokens, but the pattern thus far is use application-level APIs